src/dotnetcore/finalize/finalize.go

```diff
@@ -15,8 +15,8 @@ import (
 )
 
 var stackToRuntimeRID = map[string]string{
-	"cflinuxfs3": "ubuntu.18.04-x64",
-	"cflinuxfs4": "ubuntu.22.04-x64",
+	"cflinuxfs3": "linux-x64",
+	"cflinuxfs4": "linux-x64",
 }
```

Comment:
Do we still need this map if all the stacks use the same RID?

Reply:
I kept the map because we appear to also use it as a check to see that you are running on a compatible stack.